prow/jobs/jobs_config.yaml

```diff
@@ -0,0 +1,23 @@
+#This config is used to generate 'jobs.yaml' file from 'jobs_factory.py' using jinja2 templates.
+
+# specify which kind of jobs are present
```

Comment:
Could you clarify what it means for it to be "present"? I imagine in the future teams would want to run custom integration/canary/soak tests that we would need to expand out.

Reply:
Oh, it is just which kind of jobs out of periodics/postsubmits/presubmits are present.
I added this so that the auto-generation script does not emit an empty "presubmits:" in jobs.yaml, but I can handle it better in the script logic itself.

I will update this part.